python/shotgun_desktop/startup.py

```diff
@@ -155,12 +176,30 @@ def __initialize_sgtk_authentication(sgtk, app_bootstrap):
     :param app_bootstrap: The application bootstrap instance.
     """
 
+    # if we're importing a core that supports logging, tear down our logging
+    # to use the new one. The initial release of 0.18 didn't include the initialize_base_file_handler_from_path,
+    # so test for that as well.
+    if __is_logging_supported(sgtk):
+        logger.debug("Found a core that supports logging.")
+        # Tear down the logging
+        app_bootstrap.tear_down_logging()
```

Comment:
This is the first step mentioned in the pull request: switch from bootstrap.py's logging to core's.